### DISCUSSION TITLE
External links open in new tab

### DISCUSSION THREAD

**Discussion by thallam** [roam-garden/i#48] [Feature requests & Ideas]
It would be nice if external links opened in a new tab rather than taking the viewer away from our garden. 
This is very simple to do in HTML but cannot be done in CSS.

https://stackoverflow.com/questions/17281486/add-target-blank-in-css

**Comment by Stvad** [comment]
I'm conflicted on this one =\ . I don't like when websites do that and really prefer to control that on my own. I imagine making it a setting should be possible though

**Reply by thallam** [reply]
The default behaviour when clicking a link is different and unique on roam.garden to other web sites. 
On roam.garden - users are generally expecting a link to open up and still be able to see the page they clicked on. 
Yes, there is a symbol to indicate that the link is external but they may not all know about that or always see it and know to ctrl+click. 

Making it a setting would be great!
